loops-and-arrays.js

```diff
@@ -42,6 +55,11 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+for(let i = 0; i < squares.length; i++) {
+    let volume = squares[i] * squares[i] * squares[i]
```

Comment:
; aan  het eind, dit is de laatste die ik noem. Ik ga het niet telkens herhalen voelt als zeuren